static/app/views/dashboards/widgets/widget/description.tsx

```diff
@@ -6,28 +6,28 @@ import {IconInfo} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 
-export interface WidgetDescriptionProps {
+export interface DescriptionProps {
   description?: React.ReactElement | string;
   revealTooltip?: 'hover' | 'always';
   title?: string;
 }
 
-export function WidgetDescription(props: WidgetDescriptionProps) {
+export function Description(props: DescriptionProps) {
```

Comment:
Minor nit, but I'm not a fan of exporting generic namings. It's true that it is namespaced in a folder, but it still ends up polluting autocompletes. I see you renamed it, why not keep this like it was before :)?

Reply:
🤔 I felt that `Widget.WidgetDescription` is redundant, but I'm not opposed to it! I can namespace it for autocomplete sake